runner/templates/configmap.yaml

```diff
@@ -64,7 +64,8 @@ data:
       {{ if .Values.cache.enabled -}}
       [runners.cache]
         Type = {{ .Values.cache.type | quote }}
-        ServerAddress = "{{ template "minio.fullname" . }}:9000"
+        ServerAddress = "{{ template "minio.fullname" . }}-svc:9000"
```

Comment:
I totally agree with you.
I don't like the idea of having the port hardcoded. Moreover I  I'm thinking about completely removing `minio.fullname` template. I created a ticket for that #4.
After removing the template option port will be required part of the  `.Values.cache.serverAddress`.
But this requires a testing of the gitlab chart and it's much bigger than this pull request.